src/current/releases/v2.1.md

```diff
@@ -8,16 +8,34 @@ docs_area: releases
 keywords: gin, gin index, gin indexes, inverted index, inverted indexes, accelerated index, accelerated indexes
 ---
 
-{% assign rel = site.data.releases | where_exp: "rel", "rel.major_version == page.major_version" | sort: "release_date" | reverse %}
+<a name="v2-1-0"></a>
+<a name="v2-1-1"></a>
+<a name="v2-1-2"></a>
+<a name="v2-1-3"></a>
+<a name="v2-1-4"></a>
+<a name="v2-1-5"></a>
+<a name="v2-1-6"></a>
+<a name="v2-1-7"></a>
+<a name="v2-1-8"></a>
+<a name="v2-1-9"></a>
+<a name="v2-1-10"></a>
+<a name="v2-1-11"></a>
+<a name="v2-1-0-rc-1"></a>
+<a name="v2-1-0-rc-2"></a>
+<a name="v2-1-0-beta-20180827"></a>
+<a name="v2-1-0-beta-20180904"></a>
+<a name="v2-1-0-beta-20180910"></a>
+<a name="v2-1-0-beta-20180917"></a>
+<a name="v2-1-0-beta-20180924"></a>
+<a name="v2-1-0-beta-20181001"></a>
+<a name="v2-1-0-beta-20181008"></a>
+<a name="v2-1-0-beta-20181015"></a>
+<a name="v2-1-0-alpha-20180416"></a>
+<a name="v2-1-0-alpha-20180507"></a>
+<a name="v2-1-0-alpha-20180604"></a>
+<a name="v2-1-0-alpha-20180702"></a>
+<a name="v2-1-0-alpha-20180730"></a>
 
-{% assign vers = site.data.versions | where_exp: "vers", "vers.major_version == page.major_version" | first %}
+This release is no longer supported. For more information, see our [Release support policy]({% link releases/release-support-policy.md %}). 
 
-{% assign today = "today" | date: "%Y-%m-%d" %}
-
-{% include releases/testing-release-notice.md major_version=vers %}
-
-{% include releases/whats-new-intro.md major_version=vers %}
-
-{% for r in rel %}
-{% include releases/{{ page.major_version }}/{{ r.release_name }}.md release=r.release_name release_date=r.release_date %}
-{% endfor %}
+To download the archived documentation for this release, see [Archived Documentation]({% link releases/archived-documentation.md %}).
```

Comment:
When I look at [the rendered preview of the 'Archived Documentation' page](https://deploy-preview-21195--cockroachdb-docs.netlify.app/docs/releases/archived-documentation) v2.1 is not there.  Do you still need to make the zip archive and upload it and then edit that page to add a link?  Or were you planning to do that in a separate PR?